package.json

```diff
@@ -3,11 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.34",
```

Comment:
All of this will be added to the bundle. Which will mean the initial load is going to be very high. Are you sure you require all of `fortawesome` libs?

Reply:
MUI can be used here instead?

Reply:
My suggestion was to just include one or two or few icons instead of the whole library since the bundle will increase in size a lot. Try to use independent icons. 😄